src/rasterstats/main.py

```diff
@@ -47,7 +47,8 @@ def gen_zonal_stats(
         raster_out=False,
         prefix=None,
         geojson_out=False, 
-        boundless=True, **kwargs):
+        boundless=True,
+        cast_to_int64=None, **kwargs):
```

Comment:
Can you make `cast_to_int64` default to True to keep backwards compatibility and avoid the potential for int overflows?

Reply:
As I see it this already keeps backwards compatibility.  cast_to_int64, if it remains unspecified, will be set to `True` on 64 bit systems on line 139 of this file.

On 32 bit systems it is set to `False`.  This is exactly the same casting behaviour that existed prior to this PR